ote/src/cljs/ote/ui/service_calendar.cljs

```diff
@@ -85,18 +85,18 @@
 (defn- month-name [month]
   ;; FIXME: use translation
   (case month
-    1 "Tammi"
-    2 "Helmi"
-    3 "Maalis"
-    4 "Huhti"
-    5 "Touko"
-    6 "Kesä"
-    7 "Heinä"
-    8 "Elo"
-    9 "Syys"
-    10 "Loka"
-    11 "Marras"
-    12 "Joulu"))
+    1 (tr [:route-wizard-page :short-month-names :JAN])
```

Comment:
This is a common UI widget, I think we should use a more "common" place for the translations, not under `:route-wizard-page`.

Better yet, I think we can use JS Dates formatting:
`(.toLocaleString (doto (js/Date.) (.setMonth 0)) "fi" #js {:month "short"})` (returns "tammi")